src/apps/AppCenter/InstalledApps/AppContent.js

```diff
@@ -27,137 +29,149 @@ const AppContent = React.memo(({ repo, repoVersions, onRequestUpgrade }) => {
+          justify-content: space-between;
+          width: 100%;
+          flex-direction: ${compact ? 'column' : 'row'};
+        `}
+      >
+        <DetailsGroup compact={compact}>
+          <Heading2>Description</Heading2>
+          <div>{description}</div>
+
+          {!!repoDetails && (
+            <React.Fragment>
+              <Heading2>Details</Heading2>
+              <Markdown text={repoDetails} />
+            </React.Fragment>
+          )}
+        </DetailsGroup>
+        <DetailsGroup compact={compact}>
+          <Heading2>Installed instances</Heading2>
+          {instances.map(({ proxyAddress }) => (
             <div
+              key={proxyAddress}
               css={`
-                display: flex;
-                flex-direction: column;
-                justify-content: space-between;
-                width: 100%;
-
-                flex-direction: ${compact ? 'column' : 'row'};
+                & + & {
+                  margin-top: ${2 * GU}px;
+                }
               `}
             >
-              <DetailsGroup compact={compact}>
-                <Heading2>Description</Heading2>
-                <div>{description}</div>
-                <Heading2>Source code</Heading2>
-                <div>
-                  {sourceUrl ? (
-                    <SafeLink href={sourceUrl} target="_blank">
-                      {sourceUrl}
-                    </SafeLink>
-                  ) : (
-                    'No source code link.'
-                  )}
-                </div>
-              </DetailsGroup>
-              <DetailsGroup compact={compact}>
-                <Heading2>Installed instances</Heading2>
-                {instances.map(({ proxyAddress }) => (
-                  <div
-                    key={proxyAddress}
-                    css={`
-                      & + & {
-                        margin-top: ${2 * GU}px;
-                      }
-                    `}
-                  >
-                    <LocalIdentityBadge entity={proxyAddress} />
-                  </div>
-                ))}
-                {repoVersions && (
-                  <div
-                    css={`
-                      margin-top: ${2 * GU}px;
-                    `}
-                  >
-                    {repoVersions}
-                  </div>
-                )}
-              </DetailsGroup>
+              <LocalIdentityBadge entity={proxyAddress} />
             </div>
+          ))}
+
+          <Heading2>Source code</Heading2>
+          <div>
+            {sourceUrl ? (
+              <StyledLink href={sourceUrl}>{sourceUrl}</StyledLink>
+            ) : (
+              'No source code link.'
+            )}
           </div>
-        )
-      }}
-    </Viewport>
+
+          {!!repoName && (
+            <React.Fragment>
+              <Heading2>Package Name</Heading2>
+              <StyledLink
+                href={`https://etherscan.io/address/${repoName}#code`}
```

Comment:
Let's go to the transactions, and make sure to use [`blockExplorerUrl()`](https://github.com/aragon/aragon-ui/blob/master/src/utils/web3.js#L113) instead so it changes based on the network.

Reply:
Done 😄